### PR TITLE
Added C# official client and other minor changes

### PR DESCRIPTION
**Pull request by prashant-shahi**
- Added C# official client
- removed mention of python 2.x
- changed mention of Dgraph version from 1.0.x to 1.1.x

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/4568)

 
 
 
 

Docs Preview: [Dgraph Preview](https://dgraph-313c8eaadd-51794.surge.sh)

### CODE REVIEW

**Comment by sleto-it** [comment]
Thanks for working on this!

Please do not merge until we are fully ready to declare it officially supported (e.g. until the main _child_ issues of the _epic_ are solved as well)

Do we want to have it as first in the list (as it is now) or move it at the bottom (after Python) and keep Go as first client as it is in https://docs.dgraph.io/clients/. Probably this second option?

And hey - the preview feature rocks!

**Comment by MichaelJCompton** [comment]
Looks fine ... only is is I'd expect that by the time we drop this it'll be supporting Dgraph 1.2.0

File: wiki/content/clients/index.md [hunk 1]
```diff
@@ -439,6 +439,14 @@ fmt.Println(string(resp.Json))
...
 ```
 
+## C\#
+
+The official C# client [can be found here](https://github.com/dgraph-io/dgraph.net).
+Follow the instructions in the [README](https://github.com/dgraph-io/dgraph.net#readme) to get
+it up and running.
+
+More details on the supported versions can be found at
+[this link](https://github.com/dgraph-io/Dgraph-dotnet#supported-versions).
```
Comment (sleto-it) [inline wiki/content/clients/index.md]: this one needs to be changed as well. New link: https://github.com/dgraph-io/dgraph.net#supported-versions

Comment (prashant-shahi) [reply wiki/content/clients/index.md]: Thanks, @sleto-it.

**Comment by sleto-it** [comment]
Thanks. Needs backport to the 20.03 documentation.
The `dgraph.net` repository is still private, though. I'd suggest we do not backport before it becomes public

**Pull request merged**